### ISSUE TITLE
Why is there no split/val.txt?

### ISSUE THREAD

**Issue by city945** [LiDAR-Perception/LiDAR-CS#6]
I found this expression in the paper, "In our experiments, all the detectors are trained independently on the training split (2,000 frames of them are used as validation set during the training) of each point cloud group and evaluated on the testing split of all the groups, respectively."
However, there are only train.txt and test.txt in the split directory. I want to know if the last 2000 frames of data in train.txt are used as validation set.

**Comment by fangjin-cool** [comment]
Apologies for the delayed response. The original GitHub account could not be retrieved due to issues with two-factor authentication. In actual experiments, I randomly selected 2,000 frames from the training set as the validation set. I have performed this random selection multiple times, and the results were consistent. I recommend using random sampling.

**Comment by city945** [comment]
> Apologies for the delayed response. The original GitHub account could not be retrieved due to issues with two-factor authentication. In actual experiments, I randomly selected 2,000 frames from the training set as the validation set. I have performed this random selection multiple times, and the results were consistent. I recommend using random sampling.

"Thank you for your answer, I understand now. Additionally, do you have any plans to create a sequence-based dataset similar to NuScenes? Sometimes, sequence data can be very useful."

**Comment by fangjin-cool** [comment]
> > Apologies for the delayed response. The original GitHub account could not be retrieved due to issues with two-factor authentication. In actual experiments, I randomly selected 2,000 frames from the training set as the validation set. I have performed this random selection multiple times, and the results were consistent. I recommend using random sampling.
> 
> "Thank you for your answer, I understand now. Additionally, do you have any plans to create a sequence-based dataset similar to NuScenes? Sometimes, sequence data can be very useful."

Thank you for your suggestion, the sequential dataset is indeed very useful. Our method supports the generation of sequential data, but in LiDAR-CS, we will not release the sequential dataset for the time being, mainly because multi-sensor data with sequence information poses potential risks in terms of related data regulations.
